database/seeders/GameSeeder.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use App\Models\Game;
+
+class GameSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        Game::create(
+            ['title'=>'Arkham Horror', 'complexity_rating'=>3.58, 'release_year'=>2005, 'publisher'=>'FFG', 'category'=>'FANT, SCIF', 'playing_time_minutes'=>240, 'min_number_players'=>1, 'max_number+players'=>8],);
```

Comment:
when arrays get long like this i add a newline after each key-value pair to make it easier to read